vizro-ai/tests/unit/vizro-ai/dashboard/_response_models/test_layout.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import vizro.models as vm
+from vizro_ai.dashboard._pydantic_output import _get_pydantic_model
+from vizro_ai.dashboard._response_models.layout import LayoutPlan, _convert_to_grid
+
+
+class TestLayoutPlan:
+    """Test layout creation."""
+
+    def test_structured_output_layout_create(self, fake_llm_layout, layout_description, layout):
+        result = _get_pydantic_model(
+            query=layout_description, llm_model=fake_llm_layout, response_model=vm.Layout, df_info=None
+        )
+        assert result.dict(exclude={"id": True}) == layout.dict(exclude={"id": True})
+
+    def test_layout_plan(self, layout):
+        layout_plan = LayoutPlan(
+            layout_grid_template_areas=["graph card"],
+        )
+        result = layout_plan.create(["graph", "card"])
+        assert result.dict(exclude={"id": True}) == layout.dict(exclude={"id": True})
+
+
+@pytest.mark.parametrize(
```

Comment:
Love those tests! Ideally they could be part of the create testing no?

Reply:
Added the tests to `TestLayoutCreate` and tested the `_convert_to_grid` through `.create()`.